File: tests/conformance/fixtures/python/16-long-chain.py
from sykli import Pipeline

p = Pipeline()

p.task("A").run("echo A")
p.task("B").run("echo B").after("A")
p.task("C").run("echo C").after("B")
p.task("D").run("echo D").after("C")
p.task("E").run("echo E").after("D")

p.emit()

File: tests/conformance/fixtures/go/16-long-chain.go
package main

import sykli "github.com/yairfalse/sykli/sdk/go"

func main() {
	p := sykli.New()

	p.Task("A").Run("echo A")
	p.Task("B").Run("echo B").After("A")
	p.Task("C").Run("echo C").After("B")
	p.Task("D").Run("echo D").After("C")
	p.Task("E").Run("echo E").After("D")

	p.Emit()
}
